eth_utils/applicators.py

```diff
@@ -26,7 +37,7 @@ def apply_formatter_at_index(formatter, at_index, value):
             yield item
 
 
-def combine_argument_formatters(*formatters):
+def combine_argument_formatters(*formatters: List[Callable[..., Any]]) -> None:
```

Comment:
Correct me if I'm wrong.  These are now type hinted because I said in my previous comment "can we configure mypy to disallow code without type hints".

I wrote that under the incorrect assumption that you had completed adding type hints to the package and that it was only this new code that needed type hints.  Can you extract these extra type hint changes to their own pull request?